pkg/crc/config/settings.go

```diff
@@ -55,8 +55,13 @@ func RegisterSettings(cfg *Config) {
 	}
 
 	// Preset setting should be on top because CPUs/Memory config depend on it.
-	cfg.AddSetting(Preset, string(preset.OpenShift), validatePreset, RequiresDeleteAndSetupMsg,
-		fmt.Sprintf("Virtual machine preset (alpha feature - valid values are: %s or %s)", preset.Podman, preset.OpenShift))
+	if runtime.GOOS == "darwin" && runtime.GOARCH == "arm64" {
+		cfg.AddSetting(Preset, string(preset.Podman), validatePreset, RequiresDeleteAndSetupMsg,
+			fmt.Sprintf("Virtual machine preset (alpha feature - valid values are: %s or %s)", preset.Podman, preset.OpenShift))
```

Comment:
so this means `OpenShift` is a valid option, as that is reported in the valid values.

Reply:
validatePreset actually parse it so we might change it to just having podman, even on validatePreset side we already make aware to the user in same PR.

Reply:
yeah, not sure why I implemented this this way. Not showing the option at all sounds better.